### DISCUSSION TITLE
Is PeoplePicker component coming?

### DISCUSSION THREAD

**Discussion by siilianttivermas** [microsoft/fluentui#29941] [Q&A] [answered]
Hi, I'm trying to make a similar component which was NormalPeoplePicker in version 8. The thing that is not possible to do with ComboBox is to have those tag's of selected items inside the input element. This image from version 9 site suggest that it would be possible but it is not. Is the functionality coming perhaps in the future? This kind of people picking is really used feature in our software.
<img width="792" alt="image" src="https://github.com/microsoft/fluentui/assets/131360437/4cbf6301-93b6-4a55-9253-9d98e0cb8daa">

**Comment by spmonahan** [comment, marked as answer]
`PeoplePicker` is on the road map.

You can check out the [issue](https://github.com/orgs/microsoft/projects/786?pane=issue&itemId=24403088) for more details.
